### PR TITLE
REF: Shuffle around responsibilities for archiver data paths

### PR DESCRIPTION
**Pull request by ebolyen**

### CODE REVIEW

**Comment by ebolyen** [comment]
We will need the `relative=False` keyword for qiime-studio. @jairideout @gregcaporaso can one of you review?

**Comment by jairideout** [comment]
reviewing now

File: qiime/sdk/tests/test_visualization.py [hunk 1]
```diff
@@ -369,6 +369,27 @@ def test_get_index_paths_multiple_html_from_data_dir(self):
         with self.assertRaises(ValueError):
             visualization.get_index_paths()
 
+    def test_get_index_path_relative_false(self):
+        data_dir = os.path.join(self.test_dir.name, 'multi-html-viz2')
+        os.mkdir(data_dir)
+        multi_html_viz(data_dir, [1, 42])
+
+        visualization = Visualization._from_data_dir(data_dir,
```
Comment (jairideout) [inline qiime/sdk/tests/test_visualization.py]: this is shadowed below, I think you can remove this line and all previous lines in the method

File: qiime/sdk/tests/test_visualization.py [hunk 2]
```diff
@@ -369,6 +369,27 @@ def test_get_index_paths_multiple_html_from_data_dir(self):
         with self.assertRaises(ValueError):
             visualization.get_index_paths()
 
+    def test_get_index_path_relative_false(self):
```
Comment (jairideout) [inline qiime/sdk/tests/test_visualization.py]: `test_get_index_paths_relative_false`

**Comment by jairideout** [comment]
Had a couple test-related comments otherwise this is GTG!

**Comment by ebolyen** [comment]
Thanks @jairideout!

**Pull request merged**